HomeWorks/HW9/app.py

```diff
@@ -0,0 +1,28 @@
+from flask import Flask
+from flask_cors import CORS
+from fruits import fruits
+from vegetables import vegetables
+
+app = Flask(__name__)
+app.config.from_object(__name__)
+app.register_blueprint(fruits)
+app.register_blueprint(vegetables)
+
+
+
+# configuration
+DEBUG = True
+
+# enable CORS
+CORS(app)
+
+
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+
+
+if __name__ == '__main__':
+    print(app.url_map)
+    app.run()
```

Comment:
в тебе є вище опція `DEBUG = True`. Її можна прибрати, а замість цього задати дебаг мод як параметр
```suggestion
    app.run(debug=True)
```